omnigibson/prims/rigid_prim.py

```diff
@@ -183,10 +183,12 @@ def update_meshes(self):
                     # We need to translate the center of mass from the mesh's local frame to the link's local frame
                     local_pos, local_orn = mesh.get_local_pose()
                     coms.append(T.quat2mat(local_orn) @ (com * mesh.scale) + local_pos)
-                    # If we're not a valid volume, use bounding box approximation for the underlying collision approx
-                    if not is_volume:
-                        log.warning(f"Got invalid (non-volume) collision mesh: {mesh.name}")
-                        mesh.set_collision_approximation("boundingCube")
+                    # TODO: This is an issue for collision detection because the collision mesh being checked for collisions is the underlying mesh while
```

Comment:
We need to reenable this feature - it's a problem + there might temporarily be more objects we need to do this for as Isaac Sim is now more verbose about what happens with collision mesh baking for GPU.

Reply:
so what's the issues exactly of the TODO? Setting the boundingCube approximation gives bad values (i.e.: overapproximation of collisions)?

This seems like an asset issue. If a mesh is large and is not volume, it should be fixed at that point in time rather than resulting in a bad overapproximation downstream where we are forced to set its approximation to be boundingCube because it's not a volume, right?

Reply:
Or apply this fix in a more limited scope. suggested by @cremebrule

Reply:
Maybe lets make the overlap call also use a bounding cube in this case.

Reply:
Currently reverted. Will see if anything gets horribly broken.